week-7/Homework/mandatory/0-thinking-like-a-programmer/task.md

```diff
@@ -18,20 +18,52 @@ The second skill, problem solving, is the most important one. If you get good at
 
 Take a look at the following resources to learn more about problem solving as applied to programming:
 
-[How to THINK like a programmer](https://www.youtube.com/watch?v=NNazO2tMHno)
-[Lessons in problem solving](https://www.freecodecamp.org/news/how-to-think-like-a-programmer-lessons-in-problem-solving-d1d8bf1de7d2/)
-[Computational thinking](https://www.youtube.com/watch?v=qbnTZCj0ugI)
+- [How to THINK like a programmer](https://www.youtube.com/watch?v=NNazO2tMHno)
+- [Lessons in problem solving](https://www.freecodecamp.org/news/how-to-think-like-a-programmer-lessons-in-problem-solving-d1d8bf1de7d2/)
+- [Computational thinking](https://www.youtube.com/watch?v=qbnTZCj0ugI)
 
 After you've watched these videos I'd like you to answer these questions
 
 ## 1. What do you think the most important quality for a programmer is?
+problem solving..
+
 
 <!-- Write your answer here -->
 
 ## 2. When trying to solve a challenge, what should you do first?
+read and re-read problem untill you completly understand.
```

Comment:
Some great points here - it is very to easy to mis-read something so it is great practice to go back and double check your problem. And I always find that drawing a diagram helps me to figure things out.